src/components/ChatView.vue

```diff
@@ -134,7 +134,7 @@ export default {
 			return getTalkConfig(this.token, 'attachments', 'allowed') && this.$store.getters.getUserId()
 				&& this.$store.getters.getAttachmentFolderFreeSpace() !== 0
 				&& (this.conversation.permissions & PARTICIPANT.PERMISSIONS.CHAT)
-				&& (!hasTalkFeature(this.token, 'federation-v1') || !this.conversation.remoteServer)
+				&& !this.conversation.remoteServer // 'federation-v1' - no attachments support
```

Comment:
Does it mean (from the comment) that in `federation-2` there is no `remoteServer`?